app/keepers/modules.go

```diff
@@ -268,8 +269,8 @@ var maccPerms = map[string][]string{
 }
 
 var BeginBlockers = []string{
-	epochstypes.ModuleName,
 	upgradetypes.ModuleName,
+	epochstypes.ModuleName,
```

Comment:
Epochs being first led to delayedack and incentives trying to access new params before they have been set in migration